### DISCUSSION TITLE
Adding scss files

### DISCUSSION THREAD

**Discussion by criminal-jelly** [deanwiseman/DartSassBuilder#39] [Q&A]
I have the following section in my csproj file:
`<ItemGroup>
	<SassFile Include="wwwroot/css/pages/*.scss" />
</ItemGroup>`
I expected to see all of the scss files iterated over in the build output but the results exclude any beginning with an underscore.  

My question is, should my sample code above produce the results I expect or is there documentation I can reference to correct my mistakes please?

**Comment by RadithSandeepa** [comment]
Your code snippet does not include files beginning with an underscore because Sass treats them as partial files, which are not directly compiled but are meant to be imported into other SCSS files.

If you want to include all SCSS files, including those with an underscore, you should ensure they are imported into other SCSS files that are then compiled. For example:

```
// main.scss
@import 'partials/_file1';
@import 'partials/_file2';
@import 'pages/file3';
```

Then compile main.scss instead of individual files.

For more information, you can refer [this](https://sass-lang.com/documentation/at-rules/import/)
